src/typeDefs/inputs.ts

```diff
@@ -64,6 +64,15 @@ export const inputs = gql`
     name_of_user_starts_with: String
   }
 
+  input EditVenueInput {
+    _id: ID!
+    capacity: Int!
+    name: String!
```

Comment:
These should be allowed to be `undefined`, as it is not compulsory for the user to provide them if they aren't changed.